src/main/java/com/ericsson/ei/queryservice/ProcessQueryParams.java

```diff
@@ -76,13 +80,44 @@ public JSONArray filterFormParam(JSONObject criteriaObj, JSONObject optionsObj)
             String result = "{ \"$and\" : [ " + criteria + "," + options + " ] }";
             resultAggregatedObject = processAggregatedObject.processQueryAggregatedObject(result, databaseName, aggregationCollectionName);
         }
-        LOGGER.debug("resultAggregatedObject : " + resultAggregatedObject.toString());
+        if (filterKey == null || filterKey.equals("")) {
```

Comment:
make a method of these lines and call it here.

Reply:
Done